pkg/config/config.go

```diff
@@ -4,18 +4,54 @@ package config
 
 import (
 	"fmt"
+	"reflect"
+	"strings"
 	"time"
 
+	"github.com/mitchellh/mapstructure"
```

Comment:
this is an ARCHIVED repo we cannot use it since there is no support for it going forward. What other alternatives we could use or probably write our own helper function here ?

Reply:
unless `viper` themselves use a different library for decoding, we can't replace this

Reply:
https://github.com/go-viper/mapstructure use this plz

Reply:
`Unmarshal` is still using the old lib unfortunately 🥲

Reply:
https://pkg.go.dev/github.com/spf13/viper#DecoderConfigOption

Reply:
![image](https://github.com/user-attachments/assets/72ffb598-de8e-445c-8eb8-0f11442c7b08)

Reply:
This might be an unpopular opinion, but I feel like our configuration and our CLI is not complicated enough to warrant using either of them.

Reply:
@timraymond that makes sense, but i _really_ don't want to open that can of worm 🙂

Reply:
@nddq For sure--entirely color commentary on the codebase :)